packages/dds/tree/src/feature-libraries/node-key/nodeKeyManager.ts

```diff
@@ -92,13 +109,26 @@ class MockNodeKeyManager implements NodeKeyManager {
 	}
 
 	public localizeNodeKey(key: StableNodeKey): LocalNodeKey {
-		return brand(Number.parseInt(key.substring(30), 16) as SessionSpaceCompressedId);
+		const localNodeKey = this.tryLocalizeNodeKey(key);
```

Comment:
You can use `?? fail("Key is not compressible")` to do this whole method in one line - it's the pattern preferred by much of the ST codebase.